packages/browser/src/login/oidc/redirectHandler/ImplicitRedirectHandler.ts

```diff
@@ -67,18 +66,26 @@ export class ImplicitRedirectHandler implements IRedirectHandler {
         `ImplicitRedirectHandler cannot handle [${redirectUrl}]: it is missing one or more of [id_token, access_token, state].`
       );
     }
-    const url = new URLParse(redirectUrl, true);
+
+    const url = new URL(redirectUrl);
 
     await this.tokenSaver.saveSession(
-      url.query.state as string,
-      url.query.id_token as string,
-      url.query.access_token
+      url.searchParams.get("state") as string,
+      url.searchParams.get("id_token") as string,
+      url.searchParams.get("access_token") as string
```

Comment:
ditto

Reply:
Not possible to add type guards on a Map lookup.